### PR TITLE
[test](inverted index) add an Inverted Index Testing Switch

### PR DESCRIPTION
**Pull request by zzzxl1993**
## Proposed changes

1. enable_fallback_on_missing_inverted_index and enable_match_without_inverted_index，Default is true.

### CODE REVIEW

**Comment by zzzxl1993** [comment]
run buildall

File: regression-test/conf/regression-conf.groovy [hunk 1]
```diff
@@ -24,18 +24,18 @@ defaultDb = "regression_test"
 // init cmd like: select @@session.tx_read_only
 // at each time we connect.
 // add allowLoadLocalInfile so that the jdbc can execute mysql load data from client.
-jdbcUrl = "jdbc:mysql://127.0.0.1:9030/?useLocalSessionState=true&allowLoadLocalInfile=true"
-targetJdbcUrl = "jdbc:mysql://127.0.0.1:9030/?useLocalSessionState=true&allowLoadLocalInfile=true"
+jdbcUrl = "jdbc:mysql://127.0.0.1:49037/?useLocalSessionState=true&allowLoadLocalInfile=true"
```
Comment (qidaye) [inline regression-test/conf/regression-conf.groovy:27]: Do not modify regression-conf

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 2]
```diff
@@ -2033,6 +2036,19 @@ public void setIgnoreShapePlanNodes(String ignoreShapePlanNodes) {
     })
     public boolean enableESParallelScroll = true;
 
+    @VariableMgr.VarAttr(name = ENABLE_MATCH_WITHOUT_INVERTED_INDEX, description = {
+        "关闭无索引match查询功能，不建议正式环境使用",
```
Comment (qidaye) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:2040]:
```suggestion
        "开启无索引match查询功能，建议正式环境保持开启",
```

**Comment by zzzxl1993** [comment]
run buildall

**Review by airborne12 — DISMISSED** [review]
LGTM

**Comment by airborne12** [comment]
run buildall

**Review by qidaye — APPROVED** [review]

**Review by airborne12 — APPROVED** [review]
LGTM

**Pull request merged**